src/core/Akka.Persistence.Tests/Serialization/SerializationSpec.cs

```diff
@@ -0,0 +1,56 @@
+﻿// <copyright file="SerializationSpec.cs" company="Copaco B.V.">
+//        Copyright (c) 2015 - 2017 All Right Reserved
+//        Author: Arjen Smits
```

Comment:
@Danthar indicated in Gitter chat that this was a mistake with his ReSharper template, so no worries.